.vagrant/machines/default/virtualbox/action_provision

```diff
@@ -0,0 +1 @@
+1.5:89e0e1f7-be8b-4fc5-a492-db089f53516e
```

Comment:
Lets add everything in .vagrant to the .gitignore please. These don't belong in version control